packages/jellyfish-crypto/src/aes256.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Implementation reference: https://github.com/JamesMGreene/node-aes256
+ */
+import crypto from 'crypto'
+
+const CIPHER_ALGORITHM = 'aes-256-ctr'
+
+/**
+ * Encrypt a clear-text message using AES-256 plus a random Initialization Vector.
+ * @param {Buffer} key  A passphrase of any length to used to generate a symmetric session key.
+ * @param {Buffer} data  The clear-text message or buffer to be encrypted.
+ * @returns {Buffer}
+ */
+function encrypt (key: Buffer, data: Buffer): Buffer {
+  const sha256 = crypto.createHash('sha256')
+  sha256.update(key)
+
+  const iv = crypto.randomBytes(16)
+  const cipher = crypto.createCipheriv(CIPHER_ALGORITHM, sha256.digest(), iv)
```

Comment:
I was looking through this list, it doesn't seem like it's supported

https://www.npmjs.com/package/react-native-crypto

Not sure if I am missing something.

Reply:
replace native `crypto` usage with browserify one. Not sure it will be usable for RN either.